### PR TITLE
#14: fixes case insensitive match

### PR DESCRIPTION
**Pull request by ahwagner**

### CODE REVIEW

**Comment by ahwagner** [comment]
There is no requirements file to draw from–ignore CI build failures.

File: tests/unit/test_chembl_normalizer.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+"""Test that the therapy normalizer works as intended."""
```
Comment (korikuzma) [inline tests/unit/test_chembl_normalizer.py:1]:
```suggestion
"""Test that the chEMBL therapy normalizer works as intended."""
```

File: tests/unit/test_chembl_normalizer.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import pytest
+from therapy.normalizers import ChEMBL
+from therapy.models import Drug
+from therapy.normalizers.base import MatchType
+
+
+@pytest.fixture(scope='module')
+def chembl():
+    """Create a Wikidata normalizer instance."""
```
Comment (korikuzma) [inline tests/unit/test_chembl_normalizer.py:10]:
```suggestion
    """Create a chEMBL normalizer instance."""
```

File: tests/unit/test_chembl_normalizer.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    assert len(normalizer_response.records) == 1
+    normalized_drug = normalizer_response.records[0]
+    assert normalized_drug.label == cisplatin.label
+    assert normalized_drug.concept_identifier == cisplatin.concept_identifier
+
+
+def test_case_insensitive_primary(cisplatin, chembl):
+    """Test that cisplatin term normalizes to correct drug concept
+    as a PRIMARY match.
+    """
```
Comment (korikuzma) [inline tests/unit/test_chembl_normalizer.py:49]:
```suggestion
    """Tests that cisplatin term normalizes to correct drug concept
    as CASE_INSENSITIVE_PRIMARY, NAMESPACE_CASE_INSENSITIVE matches.
    """
```

File: tests/unit/test_chembl_normalizer.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    assert normalizer_response.match_type == MatchType.PRIMARY
+    assert len(normalizer_response.records) == 1
+    normalized_drug = normalizer_response.records[0]
+    assert normalized_drug.label == cisplatin.label
+    assert normalized_drug.concept_identifier == cisplatin.concept_identifier
+
+
+def test_case_insensitive_primary(cisplatin, chembl):
+    """Test that cisplatin term normalizes to correct drug concept
+    as a PRIMARY match.
```
Comment (jsstevenson) [inline tests/unit/test_chembl_normalizer.py:48]:
```suggestion
    """Test for case-insensitive primary and namespace 
    case-insensitive matches with cisplatin fixture.
```

**Review by korikuzma — APPROVED** [review]

**Review by jsstevenson — CHANGES_REQUESTED** [review]

**Pull request merged**